roles/commcare_analytics/tasks/rabbitmq.yml

```diff
@@ -0,0 +1,105 @@
+---
+- name: Install Erlang dependency
+  apt:
+    name: erlang
+    state: present
+  tags: rabbitmq
+
+- name: Add trusted key
+  apt_key:
+    url: https://packagecloud.io/rabbitmq/rabbitmq-server/gpgkey
+    state: present
+  tags: rabbitmq
+
+- name: Add rabbitmq official apt repository
+  apt_repository:
+    repo: 'deb https://packagecloud.io/rabbitmq/rabbitmq-server/ubuntu {{ ansible_distribution_release|lower }} main'
+    state: present
+    update_cache: "{{ apt_update_cache }}"
+  tags: rabbitmq
+
+- name: Install package
+  apt:
+    name: "rabbitmq-server={{ rabbitmq_version }}"
+    update_cache: "{{ apt_update_cache }}"
```

Comment:
I was looking for the value of `apt_update_cache` in this repo but was not able to find it. Can you share where this variable is defined?

Reply:
Hey @AmitPhulera 

Its here, https://github.com/dimagi/commcare-analytics-ansible/blob/0b53e492517ca4442e7156fd0724b07e286b5454/roles/commcare_analytics/defaults/main.yml#L6

Reply:
Thanks.